modules/thunder_tonic/thunder_tonic.module

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * The module file.
+ */
+
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function thunder_tonic_theme_registry_alter(&$theme_registry) {
+  $theme_registry['status_report_general_info']['path'] = drupal_get_path('module', 'thunder_tonic') . '/templates';
+}
+
+/**
+ * Implements template_preprocess_status_report().
+ */
+function thunder_tonic_preprocess_status_report_general_info(&$variables) {
+  if (!empty($thunder_version = \Drupal::service('extension.list.module')->get('thunder')->info['version'])) {
```

Comment:
Can we simplify this, and always set the thunder.* variables? If we cannot find $thunder_verasion, we can always set it to "n/a".

This would also simplify status-report-general-info.html.twig, where we would not have to check for the existance of the thunder variables. (See https://github.com/thunder/thunder-distribution/pull/426/files#diff-18310391031910d62d7ec596fd024a8e18bab8a434f7d072ac512d938e337eaaR38)